src/test.js

```diff
@@ -19,15 +19,16 @@ function Test( settings ) {
 
 	this.testId = generateHash( this.module.name, this.testName );
 
-	this.module.tests.push({
+	this.module.tests.push( {
 		name: this.testName,
 		testId: this.testId
-	});
+	} );
```

Comment:
But they should happen in a separate PR, not in one that introduces a new feature.